Source/SwiftLintBuiltInRules/Rules/Lint/RedundantSendableRule.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule(explicitRewriter: true)
+struct RedundantSendableRule: OptInRule {
+    var configuration = RedundantSendableConfiguration()
+
+    static let description = RuleDescription(
+        identifier: "redundant_sendable",
+        name: "Redundant Sendable",
+        description: "Sendable conformance is redundant on an isolated type",
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("struct S: Sendable {}"),
+            Example("class C: Sendable {}"),
+            Example("actor A {}"),
+            Example("@MainActor struct S {}"),
+            Example("@MyActor enum E: Sendable { case a }"),
+        ],
+        triggeringExamples: [
+            Example("@MainActor struct ↓S: Sendable {}"),
+            Example("actor ↓A: Sendable {}"),
+            Example("@MyActor enum ↓E: Sendable { case a }", configuration: ["global_actors": ["MyActor"]]),
```

Comment:
Oh I think I get it, the actor has to be defined first?

Reply:
Since the rule only operates on syntax-level, it needs to have global actors defined by name. There's no other way to know if the attribute is an actor or something else. `@MainActor` is clear and perhaps also `@...Actor` but that's too vague. Global actors aren't something that changes often, so people can just add their names to the configuration. It's always a trade-off.